tests/translator/input/kinesis_stream.yaml

```diff
@@ -0,0 +1,31 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: EventSourceMapping example
+Resources:
+  HelloWorldFn:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      InlineCode: |
+
+        exports.handler = async (event) => {
+            return {
+            statusCode: 200,
+            body: JSON.stringify(event),
+            headers: {}
+            }
+        }
+      Runtime: nodejs8.10
+      Timeout: 10
+      Events:
+        Stream:
+          Type: Kinesis
+          Properties:
+            Stream: !GetAtt GreetingStream.Arn
+            MaximumBatchingWindowInSeconds: 20
```

Comment:
Can you test a `!Ref Parameter` here as well to make sure it can handle parameter references?

Reply:
Added the parameter reference and it is working as expected.
```yaml
Parameters:
  MyBatchingWindowParam:
    Type: Number
    Default: 45
    Description: parameter for batching window in seconds
Resources:
  ...
      Events:
        Stream:
            ...
            MaximumBatchingWindowInSeconds: !Ref MyBatchingWindowParam
...
```

Reply:
I don't see this in the tests